userguide/system.rst

```diff
@@ -1165,12 +1165,11 @@ These settings are described in
    |                       |           | checkbox to appear. Changes do not take effect      |
    |                       |           | until the existing reporting database is destroyed. |
    +-----------------------+-----------+-----------------------------------------------------+
-   | Confirm RRD Destroy   | checkbox  | Destroy the reporting database. Appears when        |
-   |                       |           | :guilabel:`Graph Age` or :guilabel:`Graph Points`   |
-   |                       |           | are changed. Required for changes to                |
-   |                       |           | :guilabel:`Graph Age` or :guilabel:`Graph Points`   |
-   |                       |           | to take effect.                                     |
-   +-----------------------+-----------+-----------------------------------------------------+
+
+Any changes made to :ref:`Reporting settings <reporting_options>`
+requires the report history to be cleared. A dialog is shown when a
```

Comment:
The first sentence does not make it clear that the user gets a chance to cancel the change, even though the second sentence implies it.  How about this (not great, but clearer):
```
Changes to :ref:`Reporting settings <reporting_options>`
clear the report history. To keep history with the old 
settings, cancel the warning dialog.
```